<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Z3D::ZCameraPreviewer</class>
 <widget class="QWidget" name="Z3D::ZCameraPreviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <widget class="Z3D::ZImageViewer" name="imageView"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="snapshotButton">
       <property name="text">
        <string>Snapshot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string>Grab</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="settingsButton">
       <property name="text">
        <string>Settings...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Z3D::ZImageViewer</class>
   <extends>QGraphicsView</extends>
   <header>zimageviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
